monai/losses/ssim_loss.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+from monai.utils.type_conversion import convert_to_dst_type
+
+
+class SSIMLoss(nn.Module):
```

Comment:
Please add the new loss to: https://github.com/Project-MONAI/MONAI/blob/dev/monai/losses/__init__.py

Thanks.